Dev/src/components/TopBanner.js

```diff
@@ -14,42 +15,36 @@ class TopBanner extends FontLoadingComponent {
             RightIconName, RightTextName, RightOnPress } = this.props;
 
         return (
-            <View>
-                {this.state.fontLoaded ? (
-                    <View style={styles.entireTopView}>
-                        <View style={{flex: 0.5}}/>
-                        <View style={styles.topLeftView}  >
-                            <TouchableOpacity style={{ flex: 1, flexDirection: 'row', height: 100, justifyContent: 'flex-start', alignItems: 'center' }} onPress={LeftOnPress ? () => { LeftOnPress() } : () => {}} >
-                                <Icon
-                                    name={LeftIconName}
-                                    type="font-awesome"
-                                />
-                                <Text style={styles.leftText}
-                                    onPress={LeftOnPress ? () => { LeftOnPress() } : () => {}}>{LeftTextName}</Text>
-                            </TouchableOpacity>
-                        </View>
+            <View style={styles.entireTopView}>
+                <View style={{ flex: 0.5 }} />
+                <View style={styles.topLeftView}  >
+                    <TouchableOpacity style={{ flex: 1, flexDirection: 'row', height: 100, justifyContent: 'flex-start', alignItems: 'center' }} onPress={LeftOnPress ? () => { LeftOnPress() } : () => { }} >
```

Comment:
We should eventually move all of these inline styles to the stylesheet.styles variable